src/python/grpcio_tests/tests/unit/_cython/cygrpc_test.py

```diff
@@ -113,6 +114,7 @@ def testCredentialsMetadataPluginUpDown(self):
         lambda ignored_a, ignored_b: None, b'')
     del plugin
 
+  @unittest.skipIf(platform.python_implementation() == "PyPy", 'TODO: figure out why this fails on PyPy')
```

Comment:
(1) Wrap at eighty columns.
(2) `TODO(<username>): <work item>`, although `TODO(issue NNNN): <work item>` is far preferred.